### ISSUE TITLE
Command for sorting of cites in order of appearance

### ISSUE THREAD

**Issue by janpilu** [TGM-HIT/diploma-thesis#9]
Either by adding an option that allows the writer to choose his/her preferred sorting option:

```latex
\newif\ifsort\sortfalse         % Sort cite by appearance language
\DeclareOption{sort}{\sorttrue}
```

and and by changing thesis.cls line 75 from

```latex
\else\RequirePackage[backend=bibtex]{biblatex}\fi
```
to
```latex
\else\ifsort\RequirePackage[backend=bibtex,sorting=none]{biblatex} %sorting=none makes the cites appear in order of appearence
\else\RequirePackage[backend=bibtex]{biblatex}\fi\fi
```

Or by permanently changing thesis.cls line 75 from:
```latex
\else\RequirePackage[backend=bibtex]{biblatex}\fi
```
to:
```latex
\else\RequirePackage[backend=bibtex,sorting=none]{biblatex}\fi %sorting=none makes the cites appear in order of appearence
```

**Comment by re1** [comment]
Giving the option to sort cites by occurrence instead of their name alphabetically is definitely a good idea. Would you mind adding a Pull Request for this? I would name the option `nocitesort` for now to match existing options.

**Comment by re1** [comment]
Fixed by #11
